Tests/Tester.py

```diff
@@ -0,0 +1,45 @@
+import datetime
+
+from src import HeuristicTester
+from src.heuristics import Heuristics
+
+test_file_path = "tests.txt"
+results_file_path = "results.tsv"
+
+heuristics = [Heuristics.BfsHeuristic, Heuristics.DfsHeuristic]
+
+
+def run_tests():
+    tests = []
+
+    with open(test_file_path, "r") as test_file:
+        for line in test_file:
+            if not line.startswith("#"):
+                parts = line.rstrip().split("|")
+                tests.append((parts[0], parts[1]))
+
+    with open(results_file_path, "w") as results_file:
+        results_file.write(
```

Comment:
I think you should use the built in CSV library instead of doing it by hand: https://docs.python.org/3/library/csv.html

Reply:
Will it take care of escaping possible commas in the names of articles for me?

Reply:
It can, or you can set tab as the delimiter